README.md

```diff
@@ -66,6 +66,11 @@ npm install
 
 To run tests, run the following command:
 
+### Deployment
+
+This is the online version of my app 
+https://sazzad1q2.github.io/New-Portfolio/
+
```

Comment:
- [ ] I can see you added a deployed link to your readme Good job so far but there is something your online version is missing kindly consider checking below 👇  and fix that issue
![niby](https://user-images.githubusercontent.com/60146030/221638886-e3750c6b-942c-4ca1-a14b-7ca574287361.PNG)
- kindly consider removing the horizontal scroll on both the mobile version and desktop version as this will improve the user experience and  the below requirement will be fulfilled as well 👇 
>  👉 > Check the online version of your portfolio and make sure that the page works properly.